<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

/*  ---------------------------- ALL OTHERS WIDGETS ---------------------------- */
*{
selection-background-color: rgb(67, 128, 179);
selection-color: rgb(255, 255, 255);
}

/*  ---------------------------- MAIN WINDOW, WIDGET ---------------------------- */
QMainWindow,QWidget
{
color:rgb(0,0,0);
background:rgb(255,255,255);
font: 12pt &quot;Arial&quot;;
}

/*  ---------------------------- MENU BAR ---------------------------- */
QMenuBar
{
color:rgb(120,120,120);
background:rgb(230,230,230);
}
QMenuBar::item:selected
{
color:rgb(100,100,100);
background:rgb(200,200,200);
}
/*  ---------------------------- CONTEXT MENU ---------------------------- */
QMenu
{
color:rgb(90,90,90);
background:rgb(230,230,230);
padding: 3;
}
QMenu::item:selected
{
color:rgb(90,90,90);
background:rgb(200,200,200);
}
QMenu::separator {
background:rgb(200,200,200);
height:1px;
}


/*  ---------------------------- Q TAB BAR ---------------------------- */
QTabBar::tab {
color:rgb(150,150,150);
background:rgb(240,240,240);
height:30;
width:80;
border: rgb(240,240,240);
border-width: 0 0 2px 0;
padding-left:10;
}

QTabBar::tab:selected {
background:white;
color:rgb(100,100,100);
border: solid rgb(0,150,255);
border-width: 0 0 5px 0;
}
QTableWidget QTableCornerButton::section {
background-color: rgb(255, 255, 255);
}
/*-----------------------------------------------------------------
------------------------------ LINE_EDIT --------------------------
------------------------------ TEXT BROWSER -----------------------
------------------------------ TEXT_EDIT---------------------------
------------------------------ PLAIN_TEXT -------------------------
------------------------------------------------------------------- */
QLineEdit,QTextBrowser,QTextEdit,QPlainTextEdit
{
color:rgb(20,20,20);
background-color:white;
border: solid lightgrey;
border-width: 0 0 2px 0;
border-bottom-left-radius: 5;
border-bottom-right-radius: 5;
}

QLineEdit:disabled
{
color:rgb(160, 150, 150);
background-color:rgb(255, 240, 240);
border: solid rgb(253, 14, 14);
border-width: 0 0 2px 0;
border-bottom-left-radius: 5;
border-bottom-right-radius: 5;
}

/*  ---------------------------- COMBO BOX ----------------------------*/
QComboBox
{
color:rgb(0,115,170);
background-color:rgb(255, 255, 255);
min-width: 5px;
padding: 1px 0px 1px 3px;
border: 1px solid rgb(0,115,170);
}

QComboBox:hover
{
color:rgb(0,115,170);
background-color: white;
}

QComboBox:selected
{
color:rgb(0,115,170);
selection-background-color: rgb(255, 255, 255);
}

QComboBox::drop-down
{
width: 30px;
background-color:rgb(0,115,170);
}

QComboBox::down-arrow
{
image: url(assets/UI/Icons/interface_icons/arrow_down.png);
width: 14px;
height: 14px;
}


/* -------------------------------- CHECK BOX ----------------------------------------- */
QCheckBox
{
background: rgb(255, 255, 255);
color:rgb(25, 29, 32);
padding: 6;
}
/* ----------------------------  TOOL BOX  ----------------------------  */
QToolBox::tab
{
color:darkgrey;
background:lightgrey;
}
QToolBox::tab::selected
{
color:grey;
background:rgb(250, 250,250);
}
QToolBox::tab::hover
{
color:white;
background:rgb(0,115,170);
}
/*  ---------------------------- PROGRESS BAR ---------------------------- */
QProgressBar {
color:grey;
text-align: center;
font-size:13px;
}
QProgressBar::chunk {
background:rgb(0, 193, 50);
}
/*  ---------------------------- PUSHBUTTON ---------------------------- */
QPushButton
{
border: 1px solid lightgrey;
color:white;
background:rgb(0,115,170);
min-height:30;
min-width: 50;
}
QPushButton:hover
{
border: 1px solid lightgrey;
color:white;
background:rgb(0, 120, 210);
}

QPushButton:pressed
{
border: 1px solid lightgrey;
color:white;
background:rgb(0, 53, 100);
}
/*  ----------------------------  LCD NUMBER ---------------------------- */
QLCDNumber
{
color:rgb(0,115,170);
border:2 solid rgb(100,100,100);
}
/*  ---------------------------- TABLE_LIST_TABLE ---------------------------- */
QTableView,
QTableWidget
{
alternate-background-color: rgb(240, 250, 255);
}
QTreeView
{
background: rgb(250,250,250);
color: rgb(180,180,180);
}
QTableView::item:selected, 
QListView::item:selected,
QTableView::item:hover, 
QListView::item:hover, 
QTreeView::item:hover
{
background:rgb(0,115,170);
color:rgb(250,250,250);
}
QTableView::item, 
QListView::item, 
QTreeView::item
{
color:rgb(100,100,100);
}
QTreeView::item:selected,QListView::item:selected,QTableView::item:selected
{
color:rgb(37, 62, 71);
background:rgb(209, 241, 252);
}
/* QTreeView::item:has-children
{
background-color: rgb(0, 78, 134);
color: white;
border-bottom: 2px solid qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0 rgba(255, 255, 255, 0), stop:0.5 rgba(0, 150, 255, 255), stop:1 rgba(255, 255, 255, 0));
}
*/
/*  ---------------------------- HEADER VIEW ---------------------------- */
QHeaderView::section
{
color:rgb(133, 133, 133);
background:white;
border:transparent;
text-align:center;
padding:1;
}
/* ------------------------------- CALENDAR -------------------------------------------------- */
QCalendarView
{
color: rgb(20,20,20);
background-color: rgb(240,240,240);
alternate-background-color: rgb(0,115,170);
selection-background-color: white;
selection-color: black;
}
QAbstractItemView
{
color:rgb(200,200,200);
}


/* ---------------------------------------- SLIDER HORIZONTAL ----------------------------------------------- */


QSlider::groove:horizontal,QSlider::add-page:horizontal
{
background: rgb(255, 255, 255);
height: 27px;
}
QSlider::sub-page:horizontal {
height: 10px;
background: rgb(0,115,170);
}
QSlider::handle:horizontal {
margin-right: -10px;
margin-left: -10px;
background: rgb(0,115,170);
}
QSlider::handle:horizontal:hover {
background:rgb(0,115,170);
}

/* --------------------------------  VERTICAL SLIDER --------------------------------------------------------------  */

QSlider::handle
{
border-radius: 3px;
}

QSlider::groove:vertical,QSlider::add-page:vertical,QSlider::sub-page:vertical
{
width: 20px;
background: rgb(255, 255, 255);
}

QSlider::handle:vertical {
margin-top: -10px;
margin-bottom: -10px;
background: rgb(0,115,170);
}
QSlider::handle:vertical:hover {
background: rgb(0,115,170);
}

/* --------------------------------- SCROLLBAR HORIZONTAL --------------------------------------  */

QScrollBar::groove:horizontal{
background: white;
height: 17px;
}
QScrollBar::sub-page:horizontal,QScrollBar::add-page:horizontal  {
height: 10px;
background: rgb(255, 255, 255);
}
QScrollBar::handle:horizontal {
margin-right: -5px;
background: rgb(0,115,170);
}
QScrollBar::handle:horizontal:hover {
background: rgb(0,115,170);
}

/* --------------------------------------- SCROLLBAR VERTICAL ----------------------------------------------  */

/* SCROLLBAR */
QScrollBar:vertical {
background: white;
width: 15px;
margin: 22px 0 22px 0;
}
/* HANDLE*/
QScrollBar::handle:vertical {
background: rgb(0,115,170);
min-height: 20px;
}

/* UP ARROW */
QScrollBar::up-arrow:vertical {
image: url(assets/UI/Icons/interface_icons/arrow_up.png);
width: 10px;
height: 10px;
}
/* DOWN ARROW */
QScrollBar::down-arrow:vertical {
image: url(assets/UI/Icons/interface_icons/arrow_down.png);
width: 10px;
height: 10px;
}
/* UP BUTTON */
QScrollBar::sub-line:vertical {
background: rgb(0,115,170);
height: 20px;
subcontrol-position: top;
subcontrol-origin: margin;
}
/* DOWN BUTTON */
QScrollBar::add-line:vertical {
background: rgb(0,115,170);
height: 20px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}
/* SUBPAGES - ADDPAGE */
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
background: none;
}
/* ---------------------------------- Q TOOL BAR -------------------------------------------------------- */
/* TOOLBAR REGION */
QToolBar {
background: rgb(35,40,45);
spacing: 20;

}
/* SEPARATOR */
QToolBar:separator
{
background: rgb(80, 80, 80);
height: 2;
}
/* QToolBar QToolButton { 
    width: 100px;
} */
/* -----------------------------------------------Q TOOL BUTTON------------------------------------------- */
/* BUTTON */
QToolButton
{
color: rgb(255, 255, 255);
background:rgb(35,40,45);
}

QToolButton:hover,QToolButton:pressed
{
background-color: rgb(64, 73, 82);
}

QMessageBox QLabel
{
color: red;
}
</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>160</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="placeholderText">
       <string>E-mail or Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="placeholderText">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="placeholderText">
       <string>re-type password</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
